README.md

```diff
@@ -1,73 +1,54 @@
 
-The id of the Subscription to apply the role and permissions to.
+ To learn more about Azure integration with Cloud One Conformity, check the official [Documentation](https://cloudconformity.atlassian.net/wiki/spaces/HELP/pages/165806211/Adding+an+Active+Directory)
 
-If not supplied then the access roles will be added to all the Subscriptions in the Active Directory the App
-Registration has been created within.
-
-## Running the script
-### Azure Portal Cloud Shell (Bash)
-
-1. Log in to the Azure portal using your credentials.
-2. Open a Cloud Shell bash terminal (For details, see [Cloud shell docs](https://docs.microsoft.com/azure/cloud-shell/overview)).
-3. Run the following
-```bash
-curl -s https://raw.githubusercontent.com/cloudconformity/azure-onboarding-scripts/master/apply-roles | bash /dev/stdin \
-    --application-id <App registration client id>
-```
-
-### On local terminal
-
-#### Pre-requisites
-1. Bash version >= 4
-2. [jq](https://stedolan.github.io/jq/)
-
-_Note: Both of these are available by default in the Azure Cloud Shell._
-
-#### Running the script
-1. Install Azure CLI ([Installation instructions](https://docs.microsoft.com/cli/azure/install-azure-cli?view=azure-cli-latest)).
-2. Log into your Azure account using `az login`.
-3. Clone the Github repository or copy all the files to your local machine.
-4. Make the following modifications to the bash script
-   1. Change the value of the `GITHUB_URI` variable to `"."`.
-   2. Rename all reference to the `--template-uri` parameter to `--template-file`.
-5. Locate the id of the App Registration that the custom role is to be applied to.
-6. In the same directory as the bash script run.
-```bash
-bash apply-roles --application-id <App registration client id>
-```
-
-## Known limitations
+## Contributing
 
-1. _Doesn't gracefully handle subscriptions with no permissions to update_
+If you encounter a bug, think of a useful feature, or find something confusing
+in the docs, please
+[Create a New Issue](https://github.com/cloudconformity/azure-onboarding-scripts/issues/new)!
 
-   When running the script against all subscriptions in the Active Directory if there is a subscription which the user running
-the script doesn't have permissions to apply the roles to, the script will fail. Any subscriptions that were processed before
-this subscription will have the roles applied.
+ **PS.: Make sure to use the [Issue Template](https://github.com/cloudconformity/azure-onboarding-scripts/tree/master/.github/ISSUE_TEMPLATE)**
 
-## Contributing
+We :heart: pull requests. If you'd like to fix a bug, contribute to a feature or
+just correct a typo, please feel free to do so.
 
-The code style of the shell script follows the [Google Shell Style Guide](https://google.github.io/styleguide/shellguide.html)
```

Comment:
❓ You don't think it is a good idea to follow the Google Shell Style Guide? IMO It will make it more maintainable going forward as there will be a standard for consistent code style.